process-engine/bpmn-studio
# ✨ CallActivity termination

**Changes:**

Implement the fix for CallActivity ProcessTermination from the [ProcessEngineCore package](https://github.com/process-engine/process_engine_core/pull/260).

PR: #1421

## How can others test the changes?

- Create a Process with a CallActivity.
- Create a Subprocess for that CallActivity - ideally with a suspendable task, so you can follow its progress.
- Manually stop either the subprocess, or the parent process with the CallActivity.
- See that both processes will terminate correctly.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
